devices/bosch.js

```diff
@@ -13,6 +12,26 @@ const ea = exposes.access;
 // Radiator Thermostat II
 const boschManufacturer = {manufacturerCode: 0x1209};
 
+// BMCT
+const stateDeviceType = {
+    'Light': 0x04,
+    'Shutter': 0x01,
+};
+// BMCT
+const stateMotor = {
+    'Idle': 0x00,
+    'Opening': 0x02,
+    'Closing': 0x01,
+};
+// BMCT
+const stateSwitchType = {
+    'Button': 0x01,
+    'Button_Key_Change': 0x02,
```

Comment:
Please make these snake case, e.g. `Button_Key_Change` should become `button_key_change`

Reply:
Done